modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/EditStyleBookEntryDisplayContext.java

```diff
@@ -574,14 +573,27 @@ private String _getStyleBookEntryTitle() {
 	}
 
 	private String _getThemeName() {
-		Group group = _themeDisplay.getScopeGroup();
+		long companyId = _themeDisplay.getCompanyId();
+		String themeId = _styleBookEntry.getThemeId();
```

Comment:
Let's follow the pattern from Brian's last PR review. No need to create variables for companyId or themeId, just use _styleBookEntry.getThemeId() in each place.